modules/creating-nfd-cr-web-console.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/psap-node-feature-discovery-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-nfd-cr-web-console_{context}"]
+= Create a NodeFeatureDiscovery CR by using the web console
+
+As a cluster administrator, you can create a `NodeFeatureDiscovery` CR by using the {product-title} web console.
+
+.Prerequisites
+
+* You have access to an {product-title} cluster
+* You have logged in as a user with `cluster-admin` privileges.
+* You have installed the the NFD Operator.
+
+.Procedure
+
+. Navigate to the *Operators* -> *Installed Operators* page.
+. Find *Node Feature Discovery* and see a box under *Provided APIs*.
```

Comment:
I'm not sure what the UI looks like here, but this seems like an odd step to me. Should this be combined with the next step to say something like this? "In the *Node Feature Discovery* section, under *Provided APIs*, click *Create instance*."